packages/server/modules/core/domain/streams/operations.ts

```diff
@@ -203,6 +203,10 @@ export type UserStreamsQueryParams = BaseUserStreamsQueryParams & {
    * Pagination cursor
    */
   cursor?: MaybeNullOrUndefined<string>
+  /**
+   * Fields used to sort the result
+   */
+  sortBy?: MaybeNullOrUndefined<string[]>
```

Comment:
Can we make it clear in a comment or type that:

- Only accepts `role`
- Does nothing with invalid `sortBy` values